net/pb/Makefile

```diff
@@ -10,14 +14,15 @@ deps:
 
 %.pb.go: %.proto
 	protoc \
-	--go_out=. --plugin protoc-gen-go="${GOBIN}/protoc-gen-go" \
-    --go-grpc_out=. --plugin protoc-gen-go-grpc="${GOBIN}/protoc-gen-go-grpc" \
-    --go-vtproto_out=. --plugin protoc-gen-go-vtproto="${GOBIN}/protoc-gen-go-vtproto" \
+	-I. \
+	--go_out=. --plugin protoc-gen-go="$(PROTOC_GEN_GO)" \
+    --go-grpc_out=. --plugin protoc-gen-go-grpc="$(PROTOC_GEN_GO_GRPC)" \
+    --go-vtproto_out=. --plugin protoc-gen-go-vtproto="$(PROTOC_GEN_GO_VTPROTO)" \
     --go-vtproto_opt=features=marshal+unmarshal+size \
-	$<
+	$< # This line specifies the input file
 
 clean:
 	rm -f *.pb.go
 	rm -f *pb_test.go
 
-.PHONY: clean
\ No newline at end of file
+.PHONY: clean deps 
```

Comment:
question: Why did you add `deps` here?

Reply:
I thought it was the the best practice to make sure the right target is always called. Even if it's not strictly necessary, it doesn't hurt AFAIK.
Do you have a different opinion?

Reply:
Don't we want deps to run every time? Even if we don't, shouldn't it just be added above with the deps target as a separate phony target with only deps.

More reference on phony
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
added PHONY above each target